networking_dvs/utils/dvs_util.py

```diff
@@ -170,6 +170,28 @@ def uuid(self):
                                                              self._dvs, 'uuid')
         return self._uuid
 
+    @property
+    def mtu(self):
```

Comment:
That's an API call on what should be a fast path I guess ?

Reply:
Good point. @mmidolesov, could you perhaps store that value on init?

Reply:
Ok